css-fonts-5/Overview.bs

```diff
@@ -59,7 +59,83 @@ rules defined in CSS Fonts Level 4.
 This specification is currently a delta to the CSS Fonts Level 4 specification.
 Do not assume that if something is not here, it has been dropped.
 
+<h2 id="environment-variables">
+Environment Variables</h2>
+
+Text Scale {#text-scale-envs}
+------------------------------------------------------------------
+
+<table dfn-type=value dfn-for="env()">
+	<tr>
+		<th>Name
+		<th>Value
+		<th>Number of dimensions
+	<tr>
+		<td><dfn>preferred-text-scale</dfn>
+		<td><<number>>
+		<td>0 (scalar)
+</table>
+
+The text-scale [=environment variables=] define the devices text scale, which
+must be a non-zero number for which all text can be scaled by.
+
+The preferred-text-scale unit must be a value that computes to the documents
+initial font-size, when multiplied by 16px:
+
+<pre highlight=css>
+	:root {
+		font-size: calc(16px * env(preferred-text-scale));
+	}
+</pre>
+
+<h2 id="text-scale-meta">
+Text-Scale <code class=html>&lt;meta&gt;</code> element</h2>
+
+A document with a <code>&lt;meta></code> tag whose <code>name</code> attribute
+is a <a>ASCII case-insensitive</a> match for
+<dfn lt=text-scale><code>"text-scale"</code></dfn> is recognized as setting the
+initial font size of the document. The value of the <code>content</code>
+attribute must be an <a>ASCII case-insensitive</a> match for one of the
+recognized keywords.
+
+Documents without this <code>&lt;meta></code> tag will have an assumed default
+value of <code>legacy</code>.
+
+<h3 id="text-scale-meta-keywords">
+Keywords</h3>
+
+The recognized keywords in the [=text-scale=]
+<code class=html>&lt;meta&gt;</code> element are:
+
+<ul>
+	<li><code class="index" lt="legacy!!text-scale-meta">legacy</code></li>
+	<li><code class="index" lt="scale!!text-scale-meta">scale</code></li>
+</ul>
+
+<h3 id="legacy-keyword">The 'legacy' keyword</h3>
+
+The <dfn for="text-scale" export><code>legacy</code></dfn> property is
+recognized in the [=text-scale=] content attribute value.
+
+When the value of the [=text-scale=] content attribute is
+<a for="text-scale">legacy</a> the user agent should set the initial font size
+to 16px. The user agent may chose a different initial value, such as one based
+on a user preference. The ''env()/preferred-text-scale'' value must be 1.
```

Comment:
The value is always 1 on desktop, but not necessarily on mobile.

Reply:
Does this mean in the `legacy` mode that `env(preferred-text-scale)` may not be congruent with the font-size? As in `font-size: medium` will resolve to `16px` even though the rest of the user's OS is at 24px because they have a text-scale 1.5? So an author cannot rely on `calc(16px * env(preferred-text-scale))` resolving to the same CSS pixels as `1rem`?

Reply:
That's correct, and is what happens in today's pre-`<meta name="text-scale">` world.

We expect/hope that `<meta name="text-scale" content="scale">` becomes best practice so that authors don't have to deal with today's inconsistencies, one of which is what you pointed out.